### ISSUE TITLE
GWT build fails

### ISSUE THREAD

**Issue by chancezeus** [libgdx/ashley#278]
Using the following versions:
libGdx version: 1.9.10
ashley version: 1.7.3
gwt gradle plugin version: 1.0.9
gwt version: 2.8.2

The project build fails with the following error:
```
Compiling module com.appelit.game_base.GdxDefinition
   Tracing compile failure path for type 'com.badlogic.ashley.utils.ImmutableArray'
      [ERROR] Errors in 'com/badlogic/ashley/utils/ImmutableArray.java'
         [ERROR] Line 74: Type mismatch: cannot convert from capture#1-of ?[] to V[]
   [ERROR] Aborting compile due to errors in some input files
```

My best guess is that this is caused because the method argument is of type `Class<?>` instead of `Class<V>` but I do not have experience with GWT so if anybody can shed some light on the issue it would be appreciated...

**Comment by pelmegov** [comment]
I have similar error. who know how to fix it ?

**Comment by mgsx-dev** [comment]
@chancezeus @pelmegov maybe you could try the fix in order to confirm it works fine, build steps included in #279 comments

i think you can patch it manually instead of pulling my PR locally :-)

**Comment by tommyettinger** [comment]
You can also use JitPack to depend on @mgsx-dev 's fork's branch, if you don't feel like building Ashley: https://jitpack.io/#mgsx-dev/ashley/fix~gwt-generic-SNAPSHOT .

**Comment by snrn-Pontus** [comment]
> @chancezeus @pelmegov maybe you could try the fix in order to confirm it works fine, build steps included in #279 comments
> 
> i think you can patch it manually instead of pulling my PR locally :-)

This worked for me!
